packages/twenty-website/src/app/layout.css

```diff
@@ -46,7 +46,7 @@ nav.toc li{
 
 nav.toc a{
     color: #141414;
-    font-family: 'Inter, sans-serif';
+    font-family: 'Inter, monospace';
```

Comment:
I didn't mean to commit that to the code necessarily, but to use it as a dev tip to see that Inter isn't properly loading (cf what I sent on Discord to import the font)

Reply:
I removed the extra `'` in Chrome and as you see it's monospace showing up not Inter:
 <img width="512" alt="Screenshot 2024-02-23 at 14 11 05" src="https://github.com/twentyhq/twenty/assets/6399865/b7bbface-75a5-43cf-9c52-17d56493c607">